design/decisions/0002-mqtt-topic-schema.md

```diff
@@ -0,0 +1,847 @@
+# MQTT Topic Design Decision
```

Comment:
It's a good idea to keep a trace of this study. However, it would be good to state the decision upfront in a DRTL executive summary telling that we opted for proposal 3.